src/EditorFeatures/Core/Diagnostics/AbstractDiagnosticsTaggerProvider.cs

```diff
@@ -11,30 +11,28 @@
 using Microsoft.CodeAnalysis.Shared.TestHooks;
 using Microsoft.CodeAnalysis.Workspaces;
 using Microsoft.VisualStudio.Text;
-using Microsoft.VisualStudio.Text.Editor;
 using Microsoft.VisualStudio.Text.Tagging;
 
 namespace Microsoft.CodeAnalysis.Diagnostics;
 
-internal static class DiagnosticTaggingOptionsStorage
-{
-    public static readonly Option2<bool> PullDiagnosticTagging = new(
-        "dotnet_pull_diagnostic_tagging", defaultValue: true);
-}
-
 /// <summary>
 /// Base type of all diagnostic taggers (classification, squiggles, suggestions, inline-diags).  Subclasses can control
 /// things by overriding functionality in this type.  Internally, this will switch to either a pull or push cased
 /// approach at instantiation time depending on our internal feature flag.
 /// </summary>
-internal abstract partial class AbstractPushOrPullDiagnosticsTaggerProvider<TTag> : ITaggerProvider
+internal abstract partial class AbstractDiagnosticsTaggerProvider<TTag> : ITaggerProvider
     where TTag : ITag
 {
-    private readonly ITaggerProvider _underlyingTaggerProvider;
```

Comment:
this was the top level type, that existed to switch (based on the option) and either create the pull tagger or hte push tagger.   The push tagger is completely deleted, and we have no need to switch anymore.  So the pull tagger was merged into this type.